Misc updates

bwinter opened the pull request:
I found a few things to clean up / new ideas to add while setting up for onboarding this month. Excited for feedback!

thausler786 reviewed (approved):
looks good to me

thausler786 commented on PREP_CHECKLIST.md:
```diff
@@ -20,6 +20,7 @@
 - [ ] Schedule afternoon check in (30min at 1:30).
 - [ ] Schedule retro at the end of the week (1hr at 4:00), include link to retro board: https://retros.cfapps.io/retros/onboarding-sf
 - [ ] Schedule any sessions that you know the group would be interested in -- for example, BOSH boxes and lines.
+- [ ] In the past, non Engineers have appreciated it when we blocked out their whole day.
```
maybe add why they appreciated it--because it stopped people from scheduling meetings during the day.  Also, this is generally most effective when done a few weeks out, so they have time to move those meetings or avoid them being scheduled in the first place.

bwinter replied:
Agreed, I also think more generally all calendar invites should be done a few weeks out.

thausler786 commented:
Added one comment about blocking out participants times so they don't schedule meetings.

bwinter commented:
Cool, thanks Tim.

Looks like I didn't have git solo set properly . Whoops 😭

bwinter commented:
I can't find your comment. Did you just change the code directly? @thausler786 (I don't see a new commit either.)

bwinter committed:
Don't track intelliJ .idea folder.

bwinter committed:
We now use a general facilitator channel.

bwinter committed:
Mention the Slack channel we now use.

bwinter committed:
Move the scheduling check list items to the top of prep-checklist.

- I find scheduling is something that can / should be done further out.
e.g. during the time the "further out" email is sent.

bwinter committed:
Add space for a list of onboarder contact info.

- I suspect this list of users could make creating calendar invites /
other auth related activities faster.
- I also enjoy that this stores a little history of who / when someone
took it.

bwinter committed:
CF Onboarding is real!!

bwinter committed:
Make a link when refering to a file cause it's easy mode

bwinter committed:
Create a link to the onboarding sheet so it's easier to signup.

bwinter committed:
Remove hidden numbering.

- Add check box for "improve this process" item.
- Cleaner.
- Why not check off "improved the process"? Helps people actually do it.

bwinter committed:
Add suggested timing for calendar events.

- Take some thought out of this process.
 - Nice for new folks.

bwinter committed:
Add some constructive feedback.

- Maybe this should be optional?

bwinter committed:
Add a reminder to create a Zoom link.

- Be more remote friendly.

bwinter committed:
Make communication emails consistent.

- I specifically like how these help you sheculde when to send which I
think makes things a lot easier on a new facilitator.

bwinter committed:
Add retro board password.

bwinter committed:
For clarity link to gcp projects.

bwinter committed:
Update "block out whole week" check box

- PR suggestions requested including the "why" we would do this.
